Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m

```diff
@@ -261,12 +261,20 @@ - (void)attributedLabel:(BPKTappableLinkLabel *)label didSelectLinkWithURL:(NSUR
     if ([self.delegate respondsToSelector:@selector(attributedLabel:didSelectLinkWithURL:)]) {
         [self.delegate attributedLabel:label didSelectLinkWithURL:url];
     }
+
+    if ([self.delegate respondsToSelector:@selector(tappableLabel:didSelectLinkWithURL:)]) {
```

Comment:
I feel like we should only call one of these with a pereference for `tappableLabel:didSelectLinkWithURL:`

Reply:
Like this?
```
if (new_one_is_supported) {
  do new_one
} else if (old_one_is_supported) {
  do old_one
}
```

Reply:
Yeah that's what I am thinking